.env.example

```diff
@@ -194,3 +194,4 @@ DEV_RUN_ONLY="Webpack Servers,Socket Server,Embedder,Dev Server,Flush Redis,PG M
 # PGAdmin - used only with `db:start` command
 PGADMIN_DEFAULT_EMAIL='pgadmin4@pgadmin.org'
 PGADMIN_DEFAULT_PASSWORD='admin'
+SOC2_LOGS='true'
```

Comment:
I do not like the variable name as it is not only for SOC2, these logs are valid for many different kind of audits and compliance. 

I would like to `AUDIT_LOGS`.